iele-binary.md

```diff
@@ -131,6 +131,7 @@ After interpreting the strings representing programs as a `WordStack`, it should
                       | #dasmContract ( WordStack , Int , Map, IeleName , TopLevelDefinitions, Int , Int , String ) [function, klabel(#dasmContractAux)]
  // ----------------------------------------------------------------------------------------------------------------------------------------------------
     rule #dasmContract( .WordStack, _) => #emptyCode
+    rule #dasmContract( W1 : W2 : W3 : W4 : 99 : WS, NAME) => #dasmContract(99 : #take( W1 *Int 16777216  +Int W2 *Int 65536 +Int W3 *Int 256 +Int W4, WS), NAME)
```

Comment:
This `#take` isn't counting the `99` byte. The assembler included all bytes in the length, and I think it's best to count all the data that's meant to be part of the contract.

Reply:
I am actually just fixing this :)